great_expectations/data_context/store/store.py

```diff
@@ -240,7 +244,11 @@ def get_all(self) -> list[Any]:
         for obj in objs:
             try:
                 deserializable_objs.append(self.deserialize(obj))
-            except (MarshmallowValidationError, PydanticValidationError):
+            except (
+                MarshmallowValidationError,
```

Comment:
I'm wondering if this should be a generic `Exception` catch since it's wrapping deserializing a single object. If there is a bug, we could skip everything but we have testing around this.